src/wallet/wallet.h

```diff
@@ -78,7 +78,8 @@ enum WalletFeature
     FEATURE_WALLETCRYPT = 40000, // wallet encryption
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
-    FEATURE_LATEST = 60000
+    FEATURE_HD = 129900, // Hierarchical key derivation after BIP32 (HD Wallet)
```

Comment:
Strictly speaking not all 12.99's support HD, so we may want to adjust to 130000 after the branch off.